java/pom.xml

```diff
@@ -17,7 +17,7 @@
   <parent>
     <groupId>org.apache</groupId>
     <artifactId>apache</artifactId>
-    <version>18</version>
+    <version>23</version>
```

Comment:
Oh, yes. I did for this at ORC-796, but actually missed this.
- https://github.com/apache/orc/commit/f0f2e1847348333a44aa69e1c6e878b4d76e0d8e#diff-d8225ebcfc11b480a6e4f54e183b67c3ead51635a167c106d928c2abf1f9ef66